code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -64,6 +64,7 @@
 	var/obj/item/default_storage = /obj/item/weapon/storage/backpack/dufflebag/drone //If this exists, it will spawn in internal storage
 	var/obj/item/default_hatmask //If this exists, it will spawn in the hat/mask slot if it can fit
 	var/seeStatic = 1 //Whether we see static instead of mobs
+	var/seeAlerts = 1
```

Comment:
@optimumtact Atleast I don't have to go change all those on/off's in the code to TRUE/FALSE from 1/0!